scripts/asm-installer/tests/common.sh

```diff
@@ -190,6 +190,17 @@ verify_demo_app() {
     return 0
   fi
 }
+
+send_persistent_traffic() {
+  local IP; IP="$1"
+  local PAGE
+  local COUNT
+  while true; do
+    PAGE="$(curl "${IP}" -L -N -s || echo "")"
+    COUNT="$(echo "${PAGE}" | grep -c Hipster || true)"
+    { [[ "${COUNT}" -eq 0 ]] && echo -e "Receive unexpected response from demo app:\n${PAGE}" && return 1; } || sleep 1
```

Comment:
This still works as expected if you use exit 1 instead of return 1, right? If so then I'd prefer that as it makes the intention clearer.

Reply:
Done.